### PR TITLE
Ws2 threefeatures

### PR DESCRIPTION
**Pull request by wgilling**
This feature branch encapsulates the previous three branches `ws2_collection_browse`, `ws2_call2action_cards`, and `ws2_item_page`. These branches were not based off of `develop` but instead were based off of `ws2_homepage_recent_stuff` branch.

After pulling in this code, several config files will need to be imported. These are: 
block.block.thecomplextitle.yml
image.style.card_image.yml
views.view.collections.yml
views.view.display_media.yml

Aside from these config files, there are some SCSS, theme code, and block code changes -- so simply do the `drush cr` and the following issues from the WS2 Repository Notes document should be able to be seen reviewed:

- Item page | Viewer Page - removed the title above the viewer
- Item page | sidebar - Permalink - make a button
- Item page | Share this items - _use square font awesome icons (confirmed that was already in the latest code)_ in discussions with Jordyn, these icons needed anchor title values (not to be confused with aria label) so that they act like the Footer social media icons. Hold the mouse above the icons and they should indicate "Print this item", "Email this item", "Share on Twitter", and "Share on Facebook"
- Collections browse - horizontal cards
- Collections browse - glossary like pagination
- KEEP homepage - Calls to action

### CODE REVIEW

File: web/themes/custom/asulib_barrio/templates/views/views-view-unformatted.html.twig [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+{% endif %}
+{% for row in rows %}
+  {%
+    set row_classes = [
+      default_row_class ? 'views-row',
+      'css-pick'
+    ]
+  %}
+  
+  <div{{ row.attributes.addClass(row_classes) }}>
```
Comment (elizoller) [inline web/themes/custom/asulib_barrio/templates/views/views-view-unformatted.html.twig]: what is this template being used for?

File: web/themes/custom/asulib_barrio/templates/views/views-view-summary-unformatted--collections.html.twig [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    set classes = [
+    'page-link'
+    ]
+%}
+{%
+    set li_classes = [
+    'page-item'
+    ]
+%}
+{{ options.inline ? '<nav><ul class="pagination">' : '' }}
```
Comment (elizoller) [inline web/themes/custom/asulib_barrio/templates/views/views-view-summary-unformatted--collections.html.twig]: do we need the nav?

File: web/themes/custom/asulib_barrio/templates/views/views-view-unformatted.html.twig (hunk 1 above)
Comment (wgilling) [reply web/themes/custom/asulib_barrio/templates/views/views-view-unformatted.html.twig]: I wasn't supposed to add this to this branch, but something somewhere along the line reported that the template file was missing. I think it was even a watchdog message, but I'll push again after removing it.

File: web/themes/custom/asulib_barrio/templates/rrssb-button-list.html.twig [hunk 3]
```diff
@@ -1,15 +1,19 @@
 {% set class_2_fa = {
   'email': {
     'fa-class': 'fa fa-envelope',
+    'link-title': 'Email this item',
```
Comment (elizoller) [inline web/themes/custom/asulib_barrio/templates/rrssb-button-list.html.twig]: is this used on item and collection pages?

File: web/themes/custom/asulib_barrio/scss/components/views.scss [hunk 4]
```diff
@@ -330,11 +346,42 @@
...
+.vert-image-frame {
+  background-image: linear-gradient(to bottom, #ffffff, #f9f9f9, #f3f3f3, #ededed, #e7e7e7);
+} */
+
+.node--view-mode-recent-item-teaser-prism .vert-image-frame .field-content {
+  position: relative;
+  height: 300px;
+}
+
+.node--view-mode-recent-item-teaser-prism .vert-image-frame img {
```
Comment (elizoller) [inline web/themes/custom/asulib_barrio/scss/components/views.scss]: whats this section doing?

File: web/themes/custom/asulib_barrio/scss/components/views.scss [hunk 5]
```diff
@@ -194,18 +197,35 @@
             justify-content: center;
         }
     }
+    span.views-summary {
```
Comment (elizoller) [inline web/themes/custom/asulib_barrio/scss/components/views.scss]: do we need all this views-summary styling or does the pagination stuff cover it?

File: web/themes/custom/asulib_barrio/scss/components/cards.scss [hunk 6]
```diff
@@ -25,3 +25,49 @@ extends/_cards.scss
     }
   }
 }
+
```
Comment (elizoller) [inline web/themes/custom/asulib_barrio/scss/components/cards.scss:28]: we aren't getting all of this card styling from the uds bootstrap ?

File: web/themes/custom/asulib_barrio/scss/components/views.scss (hunk 4 above)
Comment (wgilling) [reply web/themes/custom/asulib_barrio/scss/components/views.scss]: I think it was overlooked in the hundreds of changes between the several branch -- it is also in the other branch that is pending on KEEP images (recent items), so would you say to remove the section here?

File: web/themes/custom/asulib_barrio/scss/components/views.scss (hunk 5 above)
Comment (wgilling) [reply web/themes/custom/asulib_barrio/scss/components/views.scss]: Probably. I don't know. I'll just remove.

File: web/themes/custom/asulib_barrio/scss/components/views.scss (hunk 4 above)
Comment (elizoller) [reply web/themes/custom/asulib_barrio/scss/components/views.scss]: yeah this file appears to have the same changes as https://github.com/asulibraries/islandora-repo/pull/451/files#diff-ead1f9a48d15332377148fc624038daa09cc0c2a219299119154c26eedadad3b

File: web/themes/custom/asulib_barrio/scss/components/cards.scss (hunk 6 above)
Comment (wgilling) [reply web/themes/custom/asulib_barrio/scss/components/cards.scss:28]: I am not sure what "card styling" you are referring to.

File: web/themes/custom/asulib_barrio/scss/components/views.scss (hunk 4 above)
Comment (wgilling) [reply web/themes/custom/asulib_barrio/scss/components/views.scss]: good-o. I will push a new version here that no longer has it.

**Comment by wgilling** [comment]
The last commit de267e9 changes a config, so the views.view.collections.yml will need to be imported again.

**Comment by elizoller** [comment]
i don't think the config for "config/sync/block.block.thecomplextitle.yml " is quite right - i assume it was meant to remove the description from the viewer page - but now i don't get the item title either.

**Pull request merged**